### ISSUE TITLE
Im a little lost.

### ISSUE THREAD

**Issue by katz0018** [katz0018/modularity-mindfulness#1]
Hey chelle, im superrr close I just can't figure out that last few pieces so all the sizes match.

@hellachella

**Comment by hellachella** [comment]
Hey!
Try removing the center-content-vertical class and instead, use unit-content-distribute and content fill.

Here is a vid about it. 
https://www.youtube.com/watch?v=dJiImLu8AAg

**Comment by katz0018** [comment]
<img width="1407" alt="screen shot 2018-09-20 at 8 59 16 pm" src="https://user-images.githubusercontent.com/31801814/45854304-7aafb380-bd18-11e8-9063-75e52be3e45f.png">


I got this to 20% or 19% is this okay to hand in?

**Comment by hellachella** [comment]
If everything else passes, yes.

On Sep. 20, 2018 9:02 p.m., Jazmin Katz <notifications@github.com> wrote:

[screen shot 2018-09-20 at 8 59 16 pm]<https://user-images.githubusercontent.com/31801814/45854304-7aafb380-bd18-11e8-9063-75e52be3e45f.png>

I got this to 20% or 19% is this okay to hand in?

**Comment by katz0018** [comment]
Yes everything else is green, it was just those three screen shots I’ve been trying for a while to fix and I goes up rather than down lol

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
